### ISSUE TITLE
[ENH] - Add JupyterHub specific Grafana pre-configured dashboards

### ISSUE THREAD

**Issue by iameskild** [nebari-dev/nebari#1481]
### Feature description

It would be great to have these monitoring dashboard already pre-configured:
https://github.com/jupyterhub/grafana-dashboards

### Value and/or benefit

Although there are existing monitoring dashboards, these would make the first experience with monitoring potentially worthwhile and gives an idea of what is possible.

### Anything else?

Add to Nebari Docs - Grafana 101

**Comment by Adam-D-Lewis** [comment]
I'm working on this.  Some dashboards in the latest version of the jupyterhub dashboard repo (https://github.com/jupyterhub/grafana-dashboards/tree/803ffa9b458fa325946de90eee288b065de46bc9) won't work until we upgrade jupyterhub to 3.1+

**Comment by Adam-D-Lewis** [comment]
So I had to manually render the dashboards, the process was:
- git clone https://github.com/jupyterhub/grafana-dashboards
- install grafonnet locally
- create a grafana admin api key and export to the GRAFANA_TOKEN env var
- I modified deploy.py
```
def deploy_dashboard(dashboard_path, folder_uid, api, global_dash=False):
    db = build_dashboard(dashboard_path, api, global_dash)

    if not db:
        return

    db = layout_dashboard(db)
    db = populate_template_variables(api, db)

    data = {
        'dashboard': db,
        'folderId': folder_uid,
        'overwrite': True
    }
    import json
    with open(f"rendered/{db['title'].lower().replace(' ','_')}.json", 'w') as f:
        f.write(json.dumps(db, indent=4))
    # breakpoint()
    # api('/dashboards/db', data)
```
- run modified deploy.py from the cloned repo

**Comment by Adam-D-Lewis** [comment]
Some of the dashboards don't work on nebari bc we are on a old version of jupyterhub, some of the pods have different labels, etc. so we need to figure out how we want handle it.  Fork the repo?  Something else?

**Comment by Adam-D-Lewis** [comment]
Fixed by https://github.com/nebari-dev/nebari/pull/1723/commits

**Comment by pavithraes** [comment]
>I'm working on this. Some dashboards in the latest version of the jupyterhub dashboard repo ([jupyterhub/grafana-dashboards@803ffa9](https://github.com/jupyterhub/grafana-dashboards/tree/803ffa9b458fa325946de90eee288b065de46bc9)) won't work until we upgrade jupyterhub to 3.1+([jupyterhub.readthedocs.io/en/3.0.0/reference/metrics.html](https://jupyterhub.readthedocs.io/en/3.0.0/reference/metrics.html))

Re-opeining - we now have jupyterhub to 3.1+, so it's work revisiting https://jupyterhub.readthedocs.io/en/3.0.0/reference/metrics.html

**Comment by viniciusdc** [comment]
@Adam-D-Lewis this was addressed already right?

**Comment by Adam-D-Lewis** [comment]
Yes, but I still think it would be useful to relook at https://github.com/jupyterhub/grafana-dashboards and try to add all the dashboards.  We hit issues before due to being on an old version of jupyterhub or jupyterlab and we could only add a subset of the dashboards to Nebari, but we've since upgraded jupyterhub and jupyterlab so we could likely add more now.
